api/app/users/api.py

```diff
@@ -543,5 +548,55 @@ def get(self):
         comments = db.session.query(UserComment).filter(
             UserComment.event_id == args['event_id'],
             UserComment.user_id == args['user_id']).all()
-
+        
         return comments
+
+class UserReviewAPI(restful.Resource):
+    @auth_required
+    def get(self):
+        req_parser = reqparse.RequestParser()
+        req_parser.add_argument('event_id', type=int, required=True)
+        req_parser.add_argument('user_id', type=int, required=True)
+        args = req_parser.parse_args()
+        user_id = args['user_id']
+        event_id =args['event_id']
+        
+        current_user = user_repository.get_by_id(g.current_user['id'])
+        if not current_user.is_event_admin(args['event_id']):
+            return FORBIDDEN
+
+        comments = db.session.query(UserComment).filter(UserComment.event_id == args['event_id'], UserComment.user_id == args['user_id']).all()        
```

Comment:
Not every review has a comment, so you can't use it to get the reviewers.
You need to use reviews to get reviewers.

I've advised on this before. Please look at how I explained the logic to work before.